python/fate_flow/pipelined_model/migrate_model.py

```diff
@@ -34,18 +34,18 @@ def gen_model_file_path(model_id, model_version):
 
 def compare_roles(request_conf_roles: dict, run_time_conf_roles: dict):
     if request_conf_roles.keys() == run_time_conf_roles.keys():
-        varify_format = True
-        varify_equality = True
+        verify_format = True
```

Comment:
please modify your ide line terminator settings

Reply:
there is no line terminator changes. I just fixed the typo `varify`. lol

Reply:
oh, sorry, please fix it on v1.5.2, not master